tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/contact_call_to_action.html

```diff
@@ -9,15 +9,15 @@
     {% endif %}
 
     {% if value.person %}
-        <cite class="contact-cta__cite">
+        <p class="contact-cta__cite">
```

Comment:
@chris-lawton I forgot to say that I removed these `<cite>` attributes as they weren't being used correctly - see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite